include/pisa/wand_data_range.hpp

```diff
@@ -124,7 +125,7 @@ class wand_data_range {
         mapper::mappable_vector<float> const &m_block_max_term_weight;
     };
 
-    enumerator get_enum(uint32_t i) const
+    enumerator get_enum(uint32_t i, float) const
```

Comment:
What is this `float`? Where is it used?

Reply:
This is an hack we have to deal with the current design. Unfortunately we need the max_score of the term to decompress the score, so it is useful only for wand in the compressed version. I couldnt come with a better design with the current implementation. I suggest to deal with it in a separate PR which will address a more generic and better design for wand_data.